tools/bin/cloud_storage_logging_test.sh

```diff
@@ -5,8 +5,10 @@ set -e
 echo "Writing cloud storage credentials.."
 
 # S3
-export AWS_ACCESS_KEY_ID="$(echo "$AWS_S3_INTEGRATION_TEST_CREDS" | jq -r .aws_access_key_id)"
-export AWS_SECRET_ACCESS_KEY="$(echo "$AWS_S3_INTEGRATION_TEST_CREDS" | jq -r .aws_secret_access_key)"
+AWS_ACCESS_KEY_ID="$(echo "$AWS_S3_INTEGRATION_TEST_CREDS" | jq -r .aws_access_key_id)"
```

Comment:
Assign variables and export them on separate lines to avoid accidentally masking return values (if something fails)